tools/dockerfile/grpc_iwyu/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:bullseye
+
+RUN apt-get update && apt-get install -y clang-11 llvm-11-dev libclang-11-dev clang-format-11 jq git cmake python
+
+RUN git clone https://github.com/include-what-you-use/include-what-you-use.git /iwyu
```

Comment:
Can we have this just by downloading a snapsnot instead of cloning them to reduce the footprint?

Reply:
Oh I guess!

Is there some documentation somewhere to set that up?

Reply:
Actually... let's tackle that later. Right now having the docker image just build it means we can iterate a little more quickly if we need to.

Reply:
Yeah, it can be done later. Just for reference, github provide a URL to download a zip file of snapshot. something like https://github.com/include-what-you-use/include-what-you-use/archive/refs/heads/master.zip